api/notd/store/schema.py

```diff
@@ -34,7 +34,7 @@
     sqlalchemy.Column(key='imageUrl', name='image_url', type_=sqlalchemy.TEXT, nullable=True),
     sqlalchemy.Column(key='animationUrl', name='animation_url', type_=sqlalchemy.TEXT, nullable=True),
     sqlalchemy.Column(key='youtubeUrl', name='youtube_url', type_=sqlalchemy.TEXT, nullable=True),
-    sqlalchemy.Column(key='backgroundColor', name='background_color', type_=sqlalchemy.TEXT, nullable=True),
+    sqlalchemy.Column(key='backgroundColor', name='background_colour', type_=sqlalchemy.TEXT, nullable=True),
```

Comment:
undo pls. we should always use american spelling

Reply:
please dont resolve these comment. just repond done when you have actually completed them and i will resolve.